src/components/structures/MessagePanel.js

```diff
@@ -576,6 +576,7 @@ module.exports = React.createClass({
         const scrollToken = mxEv.status ? undefined : eventId;
 
         const readReceipts = this._readReceiptsByEvent[eventId];
+        const editingEnabled = SettingsStore.isFeatureEnabled("feature_message_editing");
```

Comment:
ftr the SettingsStore is slow, so deferring as much of the call as possible is ideal. For features it's usually pretty quick (because the flag is highly cachable), but just a head's up.